api/src/schema/arrivals.graphql

```diff
@@ -74,6 +74,28 @@ extend type Member {
   isArrivalsCounterForStream: [Stream!]!
     @relationship(type: "COUNTS_ARRIVALS_FOR", direction: OUT)
 }
+
+extend type Oversight {
+  graduatedBacentas: [Bacenta!]!
+    @cypher(
+      statement: "MATCH (this)-[:HAS*5]->(bacentas:Graduated) RETURN bacentas"
```

Comment:
This will return graduated bacentas which have been closed down.

You have to add :Bacenta to this. 

And also, do you have something for active graduation and vacation graduationBacentas?

Reply:
@dabick14